README.md

```diff
@@ -1,4 +1,11 @@
-﻿# SCOTT sandbox
+﻿# Branch description
```

Comment:
This modification should not be in the master repo, as mentioned by @albertohata